core/chains/evm/client/pool.go

```diff
@@ -172,6 +182,31 @@ func (p *Pool) nLiveNodes() (nLiveNodes int, blockNumber int64, totalDifficulty
 	return
 }
 
+func (p *Pool) switchToBestNode() {
```

Comment:
maybe call this `checkLease` or `updateLease` based on the other comments

Reply:
Updated in [`364663e` (#10726)](https://github.com/smartcontractkit/chainlink/pull/10726/commits/364663ef700cdae19beeb7cdbc8ffc91fd526e1f)